### PR TITLE
Use LXe L-V saturation points to extract density

### PR DESCRIPTION
**Pull request by paolafer**
This PR adds a table with the liquid-vapor saturation data for liquid xenon. The code is modified to use the LXe pressure to calculate the correct density with linear interpolation. The system is assumed to be on the L-V saturation curve.
I decided to use the pressure instead of the temperature, because I think that we have a better precision measuring the pressure. Also, only PETit geometry has been changed, because it is the one where we want to use conditions as close as possible to reality. For other geometries, the standard LXe density keeps being used.

### CODE REVIEW

File: source/materials/PetXenonProperties.cc [hunk 1]
```diff
@@ -0,0 +1,90 @@
...
+  return count;
+}
+
+
+G4double GetLXeDensity(G4double pressure)
+{
+  // Interpolate to calculate the density at a given pressure.
+  // The temperature is unique, given the pressure, and it follows the liquid-vapor
+  // saturation curve of liquid xenon.
+  G4double density = 5.324 * kg/m3;
```
Comment (carmenromo) [inline source/materials/PetXenonProperties.cc:61]: Why do you initialize the density with such a high value?

Comment (paolafer) [reply source/materials/PetXenonProperties.cc:61]: No reason, whatever value works.

File: source/materials/PetXenonProperties.cc [hunk 2]
```diff
@@ -0,0 +1,90 @@
...
+      G4double x1 = data[i][1];
+      G4double x2 = data[i+1][1];
+      G4double y1 = data[i][2];
+      G4double y2 = data[i+1][2];
+      density = y1 + (y2-y1)*(pressure-x1)/(x2-x1);
+      found = true;
+      break;
+    }
+  }
+
```
Comment (carmenromo) [inline source/materials/PetXenonProperties.cc:79]: If I use the macro example with `/Geometry/PetBox/pressure 1.26 bar` and I print here:
```
G4cout << "Density = " << density <<  G4endl;
G4cout << "Pressure = " << pressure <<  G4endl;
```

I obtain:
```
Density = 1.81998e+19
Pressure = 7.8643e+08
```

which I don't think is correct, but maybe it changed units.

Comment (paolafer) [reply source/materials/PetXenonProperties.cc:79]: You have to use the correct units. The default Geant4 unit for pressure is not bar.

**Review by carmenromo — APPROVED** [review]
The density of LXe is successfully computed from a given pressure value with the liquid-vapor saturation data provided. Approved!

**Pull request merged**